lib/logstash/inputs/gelf.rb

```diff
@@ -96,7 +96,12 @@ def udp_listener(output_queue)
       # Gelfd parser outputs null if it received and cached a non-final chunk
       next if data.nil?
 
-      event = LogStash::Event.new(LogStash::Json.load(data))
+      begin
+        event = LogStash::Event.new(LogStash::Json.load(data))
+      rescue => ex
+        @logger.warn("JSON failed to parse a message. Skipping", :exception => ex, :backtrace => ex.backtrace)
```

Comment:
In other areas with json parse failures (like the json codec), we'll still accept the message but just treat it as plain text. Does that seem like something we should do here?